test/src/sender/main.cpp

```diff
@@ -27,48 +27,65 @@ using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp_controll : public cadmium::in_port<Message_t>{};
-struct inp_ack : public cadmium::in_port<Message_t>{};
+struct input_control : public cadmium::in_port<Message_t>{};
+struct input_ack : public cadmium::in_port<Message_t>{};
 
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp_ack : public cadmium::out_port<Message_t>{};
```

Comment:
give meaningful self explanatory names to  the structures

Reply:
was already changed

Reply:
verified

Reply:
verified